katalog/nfs-subdir-external-provisioner/MAINTENANCE.md

```diff
@@ -0,0 +1,45 @@
+# `nfs-subdir-external-provisioner` Package Maintenance
+
+To prepare a new release of this package:
+
+1. Generate manifests from helm charts
+```bash
+helm repo add nfs-subdir-external-provisioner https://kubernetes-sigs.github.io/nfs-subdir-external-provisioner
+helm template --release-name nfs-subdir-external-provisioner nfs-subdir-external-provisioner/nfs-subdir-external-provisioner --namespace nfs-subdir-external-provisioner | tee deploy.yaml
+
+mkdir -p resources
+
+# Variables to track the current file and content
+current_file=""
+current_content=""
+sighup_copyright="""# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+ 
+---"""
+    
+while IFS= read -r line; do
+   if [[ $line =~ ^#\ Source ]]; then
+     # If there's already a current file being processed, save it
+     if [ -n "$current_file" ]; then
+       echo -e "$sighup_copyright\n$current_content" > "resources/$current_file"
+     fi
+     
+     # Get the last element after the last "/" in the comment for the new file name
+     current_file=$(echo $line | awk -F'/' '{print $NF}' | awk '{print $1}')
+     current_content="$line"$'\n'
+   else
+     current_content+="$line"$'\n'
+   fi
+done < "deploy.yaml"
+ 
+# Save the last processed file
+if [ -n "$current_file" ]; then
+  echo -e "$sighup_copyright\n$current_content" > "resources/$current_file"
+fi
+ 
+EOF
+```
```

Comment:
is this logic here to add the license header?

you can do this with `addlicense` (the same binary we use in CI to check that the header is present):

```bash
addlicense -c "SIGHUP s.r.l" -v -l bsd -y "2017-present" ./resources```